app/src/main/java/io/heckel/ntfy/ui/MainActivity.kt

```diff
@@ -470,20 +470,12 @@ class MainActivity : AppCompatActivity(), ActionMode.Callback, AddFragment.Subsc
         if (actionMode != null) {
             handleActionModeClick(subscription)
         } else if (subscription.upAppId != null) { // Not UnifiedPush
```

Comment:
```suggestion
        } else if (subscription.upAppId != null) { // UnifiedPush
```

If `upAppId` is `!= null`, wouldn't that _be_ unified push?

Reply:
You're right. This was part of the original code, and I didn't want to touch it. We should change it for accuracy, though